RevenueCatUI/UIKit/PaywallViewController.swift

```diff
@@ -28,10 +28,19 @@ public final class PaywallViewController: UIViewController {
     public weak var delegate: PaywallViewControllerDelegate?
 
     private let offering: Offering?
-
-    // swiftlint:disable:next missing_docs
-    public init(offering: Offering? = nil) {
+    private let displayCloseButton: Bool
+
+    /// Initialize a `PaywallViewController` with an optional `Offering`.
+    /// - Parameter offering: The `Offering` containing the desired `PaywallData` to display.
+    /// `Offerings.current` will be used by default.
+    /// - Parameter displayCloseButton: Set this to `true` to automatically include a close button.
+    @objc
```

Comment:
Added this to make sure it's available since we don't have a `RevenueCatUI` Obj-C API tester.